src/components/tabs/tabs.pw.tsx

```diff
@@ -0,0 +1,822 @@
+  ] as [boolean, number][]).forEach(([bool, width]) => {
+    test(`should render Tabs with extendedLine prop set to ${bool}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<TabsComponent extendedLine={bool} />);
+
+      const val = await getStyle(tabList(page), "width");
+      expect(parseInt(val)).toBeGreaterThanOrEqual(width - 6);
+      expect(parseInt(val)).toBeLessThanOrEqual(width + 4);
+    });
+  });
+
+  ([
+    ["off", "none", "none", "none", "none"],
+    ["on", "solid", "none", "solid", "solid"],
+    ["no sides", "solid", "none", "none", "none"],
+  ] as [TabsProps["borders"], string, string, string, string][]).forEach(
+    ([state, top, bottom, right, left]) => {
+      test(`should render Tabs with borders prop set to ${state} when positioned on top`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TabsComponent borders={state} />);
+
+        const tabText = page.getByText("Tab 1", { exact: true });
+
+        await expect(tabText).toHaveCSS("border-top-style", top);
+        await expect(tabText).toHaveCSS("border-bottom-style", bottom);
+        await expect(tabText).toHaveCSS("border-right-style", right);
+        await expect(tabText).toHaveCSS("border-left-style", left);
+      });
+    }
+  );
+
+  ([
+    ["off", "none", "none", "none", "none"],
+    ["on", "solid", "solid", "none", "solid"],
+    ["no sides", "solid", "solid", "none", "none"],
+  ] as [TabsProps["borders"], string, string, string, string][]).forEach(
+    ([state, top, bottom, right, left]) => {
+      test(`should render Tabs with borders prop set to ${state} when positioned to the left`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TabsComponent position="left" borders={state} />);
+
+        const tabText = page.getByText("Tab 1", { exact: true });
+
+        await expect(tabText).toHaveCSS("border-top-style", top);
+        await expect(tabText).toHaveCSS("border-bottom-style", bottom);
+        await expect(tabText).toHaveCSS("border-right-style", right);
+        await expect(tabText).toHaveCSS("border-left-style", left);
+      });
+    }
+  );
+
+  test("should render Tabs header with correct width set by headerWidth prop", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <TabsComponent headerWidth="440px" align="left" position="left" />
+    );
+
+    await expect(tabList(page).locator("..")).toHaveCSS("width", "440px");
+  });
+
+  ([
+    ["default", "rgb(255, 255, 255)"],
+    ["alternate", "rgb(204, 214, 219)"],
+  ] as [TabsProps["variant"], string][]).forEach(
+    ([variant, backgroundColor]) => {
+      test(`should render Tabs with variant prop set to ${variant}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<TabsComponent variant={variant} />);
+
+        await expect(tabById(page, 1)).toHaveCSS(
+          "background-color",
+          backgroundColor
+        );
+      });
+    }
+  );
+
+  [
```

Comment:
The below test is also using CHARACTERS.STANDARD, so we can move it as a const to the top of the file:
```
 const specialCharacters = [
  CHARACTERS.STANDARD,
  CHARACTERS.DIACRITICS,
  CHARACTERS.SPECIALCHARACTERS,
];
```